onnxruntime/test/providers/qnn/qnn_test_utils.h

```diff
@@ -127,12 +172,14 @@ inline NodeArg* MakeTestInput(ModelTestBuilder& builder, const TestInputDef<T>&
  * \param provider_options Provider options for QNN EP.
  * \param opset_version The opset version.
  * \param expected_ep_assignment How many nodes are expected to be assigned to QNN (All, Some, or None).
- * \param num_modes_in_ep The expected number of nodes assigned to QNN EP's partition.
+ * \param num_nodes_in_graph The expected number of nodes assigned to the final graph. If all nodes are assigned to QNN,
+ *                           then this is 1 (all fused). Otherwise, it depends on the optimizations done by the CPU EP.
```

Comment:
>then this is 1 [](start = 29, length = 14)

You set it to 1 for Argmxx test, but it doesn't mean for all other tests, right?

Reply:
Removed